code/data_structures/src/stack/infix_to_postfix/infix_to_postfix_1.cpp

```diff
@@ -0,0 +1,127 @@
+#include <iostream>
+#include <stack>
+
+class InfixToPostfix
+{
+
+public:
+
+    InfixToPostfix(const std::string &expression) : expression_(expression) { }
+
+    int getPrecedenceOfOperators(char);
+
+    std::string convertInfixToPostfix();
+
+private:
+    std::string expression_;
+
+};
+
+int InfixToPostfix::getPrecedenceOfOperators(char ch)
+{
+    if(ch == '+' || ch == '-')
+        return 1;
+    if(ch == '*' || ch == '/')
+        return 2;
+    if(ch == '^')
+        return 3;
+    else
+        return 0;
+}
+
+std::string InfixToPostfix::convertInfixToPostfix()
+{
+    std::stack <char> stack1;
+    std::string infixToPostfixExp = "";
+    int i = 0;
+    while( expression_[i] != '\0' )
```

Comment:
The space should be before, not after. Can you fix _all_ style issues please?

Reply:
Ohh Sorry, I will fix these style issues...

Reply:
Changes are done, Please review it.

Reply:
@arnavb I have done the changes, please review it.